crates/sovereign-sdk/rollup-interface/src/state_machine/da.rs

```diff
@@ -42,7 +42,7 @@ pub trait DaSpec:
     type BlockHeader: BlockHeaderTrait<Hash = Self::SlotHash> + Send + Sync;
 
     /// The transaction type used by the DA layer.
-    type BlobTransaction: BlobReaderTrait<Address = Self::Address> + Send + Sync;
+    type BlobTransaction: BlobReaderTrait<Address = Self::Address> + Send + Sync + Clone;
```

Comment:
Why?

Reply:
Because `da_data` needs to be used in multiple iterations of the loop in the `process_l1_block` inside the prover. This value is used to among others to create `StateTransitionData` for each group of sequencer commitments.